.travis.yml

```diff
@@ -9,3 +9,17 @@ before_install:
 - pod repo update
 
 script: fastlane ci
+
+after_success:
+  # generate docs
+- gem install jazzy
+- jazzy --xcodebuild-arguments -scheme,Tests --module RRemoteConfig --source-directory RRemoteConfig --podspec RRemoteConfig.podspec --exclude RRemoteConfig/Loader.swift --readme README.md
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  keep_history: true
+  local_dir: docs
+  on:
+    branch: master
```

Comment:
On Android, I have it using the `gh-pages` branch, and then GitHub builds the docs from that. You can do that if you don't want CI pushing to your master branch, but it might make things more complicated for CI setup...

Reply:
Yep, that's what the `provider: pages` bit does in the yml.
The `branch: master` means that it only does the deploy when changes are pushed to master.

Reply:
Oh, nice! I read it completely wrong.